src/views/layouts-default/TheHowItWorksLayout.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <div class="mew-component--how-it-works">
     <the-layout-header title="How it works" />
-    <mew-tabs :items="mewTabs" has-underline>
```

Comment:
why did you remove tab menu?

Reply:
Alex talked about it few weeks ago about this on Tuesday meeting and we all agreed to remove tab and show all contents on one page.